app/models/claim.rb

```diff
@@ -211,11 +211,11 @@ class Claim < ApplicationRecord
     length: {maximum: 256, message: "Email address must be 256 characters or less"}, if: -> { email_address.present? }
 
   validates :provide_mobile_number, on: [:"provide-mobile-number", :submit], inclusion: {in: [true, false], message: "Select yes if you would like to provide your mobile number"}, if: :has_ecp_or_lupp_policy?
-  validates :mobile_number, on: [:"mobile-number", :submit], presence: {message: "Enter a mobile number in the correct format, for example 07123456789"}, if: -> { provide_mobile_number == true && has_ecp_or_lupp_policy? }
+  validates :mobile_number, on: [:"mobile-number", :submit], presence: {message: "Enter a mobile telephone number, like 07700 900 982"}, if: -> { provide_mobile_number == true && has_ecp_or_lupp_policy? }
   validates :mobile_number,
     format: {
       with: /\A\+?(?:\d\s?){11}\z/,
-      message: "A mobile number must be 11 digits"
+      message: "Enter a valid mobile telephone number, like 07700 900 982"
```

Comment:
Isn't this the guideline? `Enter a telephone number, like 01632 960 001, 07700 900 982 or +44 808 157 0192` https://design-system.service.gov.uk/patterns/telephone-numbers/#if-the-telephone-number-is-not-in-the-correct-format-and-there-is-no-example